darwin/future/core/items/set_priority.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict, Optional
+
+from darwin.future.core.client import ClientCore
+from darwin.future.core.types.common import JSONType
+from darwin.future.data_objects.typing import UnknownType
+
+
+def set_item_priority(
+    api_client: ClientCore,
+    team_slug: str,
+    priority: int,
+    filters: Optional[Dict[str, UnknownType]] = None,
```

Comment:
This is not blocking at all but interested here on your thoughts on 
`filters: Optional[...] = None` vs `filters: ... = {}` Is it more pythonic to assume None's or empty objects? The former does tend to lead to requiring more typecheck guards to satisfy mypy/pylance with like `assert filters is not None` etc, which does feel more verbose than required sometimes.

Reply:
but obviously, it is a bit more clear that filters isn't required via the `Optional[...] = None` approach

Reply:
Nah, good point, I like the default in this backend code, will take this suggestion